test/integration/default/default_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'pathname'
 
+ENV['PATH'] = 'C:\\msys2\\bin;C:\\msys2\\mingw32\\bin;C:\\msys2\\usr\\bin' if os[:family] == 'windows'
```

Comment:
This is performing a really silly test and is only looking to see if any compiler at all exists - it used to just check 32-bit by default.

Sadly I can't figure out how to set environment variables at all in inspec - so I'm just turning this off for now.